lib/salus/report.rb

```diff
@@ -205,11 +213,15 @@ def publish_report(directive)
                       when 'yaml' then to_yaml
                       when 'sarif' then to_sarif(directive['sarif_options'] || {})
                       when 'sarif_diff' then to_sarif_diff
+                      when 'cyclonedx-json' then to_cyclonedx(directive['cyclonedx_options'] || {})
                       else
                         raise ExportReportError, "unknown report format #{directive['format']}"
                       end
       if Salus::Config::REMOTE_URI_SCHEME_REGEX.match?(URI(uri).scheme)
-        send_report(uri, report_body(directive), directive['format'])
+        send_report(directive,
```

Comment:
Could simply pass in directive and read `directive['format']` from there as well.